### ISSUE TITLE
CfA Fellowship Questions for CTM about handoff & sustainability

### ISSUE THREAD

**Issue by mateoclarke** [open-austin/iced-coffee#341]
**Task description:** 
- Rohan and his team put a lot of effort into prototyping the Affordable Housing Data Hub. The project has been handed off to the City's CTM dept and the project is now hidden from the public in a private repo making continued collaborations with volunteers impossible.
  - How is handoff and implementation going? The missing piece to call this project/fellowship complete is the app going live on city servers. Is there a timeline for that? What can OA do to support? If it doesn’t go live we want to understand why? 
  - We need to reach out to our contacts at CTM to answer these questions. 
- Houli has a project idea for the next fellowship application. Before we entertain new project ideas, we should complete a post-mortem and understand what worked well and what needs to change for this coming class of fellows.

**Owner:** Person who completes the task. Also the "Assignee"
Vickie 

**Supporter:** People available to help with the tas  
Projects & Tech Leads

**Outcome:** 
Help us organize and understand the task we are sharing. 

**Next Steps:** 
- [ ] Vickie team to reach out to Charles, Josh Ruddow, other City folks to ask question. (Or schedule a meeting, maybe?) 
- [x] Mateo to check in with Fredrick, Sergio, and other devs helping with the project and report back to project team about technical implementation status

**Comment by mateoclarke** [comment]
I checked in with Sergio about how their meeting went with the CTM applications team. They had a meeting and it sounds like the suggestions that were proposed won't be taken up by CTM applications. I could share more offline. TL;DR: I was disappointed to hear that ODD folks don't have a role in helping move it forward or get online. Even more reason to reach out to figure out the status.

**Comment by VictoriaODell** [comment]
Just an update here. I reached out to folks at the city for a meeting but haven't received a response and know some folks are OOO for a conference right now. 

Separately I've talked to Houli and Hashim. My [notes from there are organized here](https://docs.google.com/document/d/18FLLCotbfjVmTQNA8rMt61V9k48RdcXH8Ak6hov73j4/edit?usp=sharing).

Austin will not have a fellow this year is my current understanding. We would still like to understand the status of the project and how we might best support a fellowship in the future.
